### ISSUE TITLE
Title and subtitle don't work in README example

### ISSUE THREAD

**Issue by bgotink** [seikichi/restructured#1]
The title and subtitle are missing when I run the example in the README. The output I get is

``` json
{
  "type": "document",
  "children": [
    {
      "type": "section",
      "children": [
        {
          "type": "section",
          "children": [
            {
              "type": "paragraph",
              "children": [
                {
                  "type": "text",
                  "text": "This is a paragraph.\n"
                }
              ]
            },
            {
              "type": "bullet_list",
              "children": [
                {
                  "type": "list_item",
                  "children": [
                    {
                      "type": "paragraph",
                      "children": [
                        {
                          "type": "text",
                          "text": "This is item 1\n"
                        }
                      ]
                    }
                  ]
                },
                {
                  "type": "list_item",
                  "children": [
                    {
                      "type": "paragraph",
                      "children": [
                        {
                          "type": "text",
                          "text": "This is item 2\n"
                        }
                      ]
                    }
                  ]
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}
```

**Comment by seikichi** [comment]
Hi @bgotink,

Sorry, I fixed bugs and updated the `restructured` to 0.0.7 ([npm](https://www.npmjs.com/package/restructured)). In 0.0.7, I changed the output format slightly to have a more compatible with docutils. See [README.md](https://github.com/seikichi/restructured/blob/master/README.md).

**Comment by bgotink** [comment]
Thanks a lot! :)
